### PR TITLE
post/fivetran_slack_webhook

### PR DESCRIPTION
**Pull request by JMiocevich**

### CODE REVIEW

File: _posts/2022-06-09-fivetran-webhook-notifications.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+image: img/fivetran_logo.png
+description:
+  Creating fivetran event notifications using webhooks
+---
+
+![Fivetran Slack Notifications](/img/fivetranslacklambdalogo.png)
+
+<br/>
+
+Getting timley notifications on whether a Fivetran data connector is broken, or a dbt transformation has failed, is crucial to maintaining confidence in your datapipeline and reducing your time to restore. This blog will cover how to build your own system in AWS utilising Fivetran's newly released [webhooks](https://fivetran.com/docs/rest-api/webhooks) functionality and [slack webhooks](https://slack.com/intl/en-au/help/articles/115005265063-Incoming-webhooks-for-Slack).
```
Comment (Hamo69) [inline _posts/2022-06-09-fivetran-webhook-notifications.md:16]: perhaps link dbt to site so the user knows what it is..

File: _posts/2022-06-09-fivetran-webhook-notifications.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+<br>
+## Overview
+
+Fivetran's new webhook functionality allows you to subscribe to fivetran events, and push these to a webhook listener. Currently as this feature is still in beta, there is no way to directly connect this to slack or a incident management system, to resolve this we will be creating our own serverless webhook listener.
```
Comment (Hamo69) [inline _posts/2022-06-09-fivetran-webhook-notifications.md:20]: fivetran should be Fivetran

Comment (Hamo69) [inline _posts/2022-06-09-fivetran-webhook-notifications.md:20]: an incident management system

File: _posts/2022-06-09-fivetran-webhook-notifications.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+Currently, webhooks can only be created using the Fivetran API, for moreinfomation on Fivetran webhooks, see [documentation](https://fivetran.com/docs/rest-api/webhooks#createwebhooks)
+
+
+In PostMan set URL to 
+
+```javascript
+POST https://api.fivetran.com/v1/webhooks/account
+```
+
+Set Authorisation to Basic Basic Auth
```
Comment (Hamo69) [inline _posts/2022-06-09-fivetran-webhook-notifications.md:54]: Basic Basic Auth ?

**Review by Hamo69 — APPROVED** [review]

File: _posts/2022-06-09-fivetran-webhook-notifications.md (hunk 2 above)
Comment (JMiocevich) [reply _posts/2022-06-09-fivetran-webhook-notifications.md:20]:
```suggestion
Fivetran's new webhook functionality allows you to subscribe to Fivetran events, and push these to a webhook listener. Currently, as this feature is still in beta, there is no way to directly connect this to slack or an incident management system, to resolve this we will be creating our own serverless webhook listener.
```

File: _posts/2022-06-09-fivetran-webhook-notifications.md (hunk 3 above)
Comment (JMiocevich) [reply _posts/2022-06-09-fivetran-webhook-notifications.md:54]:
```suggestion
Set Authorisation to Basis Auth 
```

File: _posts/2022-06-09-fivetran-webhook-notifications.md (hunk 1 above)
Comment (JMiocevich) [reply _posts/2022-06-09-fivetran-webhook-notifications.md:16]:
```suggestion
Getting timley notifications on whether a Fivetran data connector is broken, or a [dbt transformation](https://fivetran.com/docs/transformations/dbt) has failed, is crucial to maintaining confidence in your datapipeline and reducing your time to restore. This blog will cover how to build your own system in AWS utilising Fivetran's newly released [webhooks](https://fivetran.com/docs/rest-api/webhooks) functionality and [slack webhooks](https://slack.com/intl/en-au/help/articles/115005265063-Incoming-webhooks-for-Slack).
```

**Pull request merged**